src/Traversal/Sdk/Sdk.targets

```diff
@@ -89,9 +89,9 @@
     <ProjectFile Remove="@(ProjectFile)" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TraversalGlobalProperties)' != ''">
     <ProjectReference Update="@(ProjectReference)"
-                      AdditionalProperties="$(TraversalGlobalProperties)" />
+                      AdditionalProperties="%(AdditionalProperties);$(TraversalGlobalProperties)" />
```

Comment:
Do you have time to write a unit test?

Reply:
I just looked into the tests for Traversal and it would probably take me an hour or more to familiarize myself with msbuild testing. I filed an issue to track this https://github.com/microsoft/MSBuildSdks/issues/161 and I hope we can get this merged meanwhile to unblock my work in dotnet/runtime and I'll hopefully have time to follow-up on the issue EOW. Does that sound good @jeffkl?

Reply:
Sounds good.  I'll add the test, thanks for filing the tracking issue.